### ISSUE TITLE
yo

### ISSUE THREAD

**Issue by ColtNovak** [om3ga6400/om3ga6400.github.io#1]
btw I found out crosh and chrome://flags is unblocked on my computer,idk about yours. what should I do with it?

**Comment by om3ga6400** [comment]
also we should make an organization @ColtNovak

**Comment by ColtNovak** [comment]
ok, but idk if that's a good idea. the other exploits were a lot better. like I think cryptosmite let you dualboot and switch between the two

**Comment by om3ga6400** [comment]
on a un-related note, do you like the website?

**Comment by ColtNovak** [comment]
yeah its cool

**Comment by om3ga6400** [comment]
we need an org so we can make it into a chat room

**Comment by om3ga6400** [comment]
don't you have a org?

**Comment by ColtNovak** [comment]
but I am looking for a way to bypass the admin blocked this site thing. but I don't want to use a slowaf proxy. can I connect to a VPN with crosh?
I have a ,tech

**Comment by om3ga6400** [comment]
idk I haven't used crosh much all though you can unblock most sites with a mobile hotspot

**Comment by ColtNovak** [comment]
but I was looking into setting up a WebCord sercer, I have temp access to Google IDX right now, I got the client to run but no web hosting and I can't get the server to run

**Comment by om3ga6400** [comment]
other than the sites that when you try to connect to them they say "x is blocked"

**Comment by om3ga6400** [comment]
is webcord a web discord server?

**Comment by om3ga6400** [comment]
try vercel and do you have a org?

**Comment by ColtNovak** [comment]
idk, I don't.
I have a .tech, and a .digital. we don't need a .org

**Comment by om3ga6400** [comment]
I meant a gh org

**Comment by om3ga6400** [comment]
organization

**Comment by ColtNovak** [comment]
ohh. nah sadly. but can you try to run a webcord server?

**Comment by ColtNovak** [comment]
wait I think its actually a spacebar server

**Comment by om3ga6400** [comment]
its going through a big rewrite rn should we wait till that done because it might just be broken rn

**Comment by ColtNovak** [comment]
nah it is fine from what I have seen.

**Comment by ColtNovak** [comment]
if not, theres something called oldcord that is cool

**Comment by ColtNovak** [comment]
https://1420-idx-deadsd-1727105865585.cluster-ux5mmlia3zhhask7riihruxydo.cloudworkstations.dev/

**Comment by om3ga6400** [comment]
requires me to use a Google account so I can't use that

**Comment by ColtNovak** [comment]
rip nvm but try to host the client in vercel rq

**Comment by ColtNovak** [comment]
well how about you try to get the server to work while I do the client

**Comment by om3ga6400** [comment]
ok

**Comment by om3ga6400** [comment]
its being deployed

**Comment by ColtNovak** [comment]
bet

**Comment by om3ga6400** [comment]
wait I thought you said you already got the client working

**Comment by ColtNovak** [comment]
slowly and on an IDE

**Comment by om3ga6400** [comment]
I got "Error: No Output Directory named "public" found after the Build completed. You can configure the Output Directory in your Project Settings."

**Comment by ColtNovak** [comment]
but does Vercel give terminal access

**Comment by ColtNovak** [comment]
follow the docs

**Comment by om3ga6400** [comment]
failed while using docs I think it broken

**Comment by om3ga6400** [comment]
I'ma try on gh codespaces

**Comment by om3ga6400** [comment]
my god this carp has a lot of unsupported packages

**Comment by om3ga6400** [comment]
got it to work

**Comment by om3ga6400** [comment]
lets setup a org to hold the repos

**Comment by om3ga6400** [comment]
what's should we name it

**Comment by om3ga6400** [comment]
@ColtNovak

**Comment by ColtNovak** [comment]
wait I was gone mb,  you got the server up?

**Comment by om3ga6400** [comment]
yes what should I name the organization that will have the repo on it

**Comment by ColtNovak** [comment]
codechat, i own codechat.tech

**Comment by ColtNovak** [comment]
https://client1-three.vercel.app/ btw

**Comment by om3ga6400** [comment]
but we aren't only gonna do the chat thing are we\

**Comment by ColtNovak** [comment]
wym?

**Comment by ColtNovak** [comment]
I just need a chatroom like this for talking to people lol

**Comment by om3ga6400** [comment]
if were gonna do more than just the chat the gh organizaion should be named something other than codechat

**Comment by om3ga6400** [comment]
we need cool name for this 
![image](https://github.com/user-attachments/assets/11d52e38-5c2f-43d6-b715-0857177687bb)

**Comment by om3ga6400** [comment]
cmon we gotta think of a good name

**Comment by om3ga6400** [comment]
I think fish dev would be funny

**Comment by ColtNovak** [comment]
bet

**Comment by ColtNovak** [comment]
btw check out https://chat.codechat.tech/

**Comment by ColtNovak** [comment]
can we make http://codechat.tech the instance of the server for the login, but just give me the current domain so I can testit

**Comment by om3ga6400** [comment]
ok lemme host it on vercel

**Comment by om3ga6400** [comment]
org https://github.com/F1sh-Dev

**Comment by ColtNovak** [comment]
that's the client but you gotta do the server on the same domain

**Comment by om3ga6400** [comment]
wdym

**Comment by om3ga6400** [comment]
how I use console in vercel

**Comment by ColtNovak** [comment]
no clue lol, you might need to do another way

**Comment by ColtNovak** [comment]
made any progress

**Comment by ColtNovak** [comment]
maybe put them as build commands?

**Comment by om3ga6400** [comment]
I hosted it on gh codespaces https://opulent-enigma-jp4r6q76564fpjg5-3001.app.github.dev/

**Comment by ColtNovak** [comment]
ok

**Comment by om3ga6400** [comment]
@ColtNovak make yourself public on the org

**Comment by ColtNovak** [comment]
ok, but my issue is that if you try to login or register it says it cant fetch the email

**Comment by om3ga6400** [comment]
why don't we just use the official server?

**Comment by ColtNovak** [comment]
cuz who wants to do that, I wanna customize it

**Comment by om3ga6400** [comment]
well if our doesn't work and it might cost money also the default server is unblocked

**Comment by om3ga6400** [comment]
it uses my codespaces hours

**Comment by ColtNovak** [comment]
aren't you pro?

**Comment by om3ga6400** [comment]
thats with pro

**Comment by ColtNovak** [comment]
oh wait what

**Comment by ColtNovak** [comment]
well if we build it in codespaces cant we switch to Vercel once we set it up

**Comment by om3ga6400** [comment]
that's the thing after I hosted it on gh codespaces there's no changes to the repo

**Comment by om3ga6400** [comment]
![image](https://github.com/user-attachments/assets/e644a8eb-e9f2-49e0-8523-e938b390dba7)

**Comment by ColtNovak** [comment]
cant we pull it?

**Comment by om3ga6400** [comment]
no I just said there no changes

**Comment by ColtNovak** [comment]
ohhh

**Comment by om3ga6400** [comment]
I think it might deploy if I set the server folder as the one to deploy in vercel because that's what I had to on the codespace

**Comment by om3ga6400** [comment]
like I had to do CD server

**Comment by ColtNovak** [comment]
yeah

**Comment by om3ga6400** [comment]
I think I got it to work

**Comment by om3ga6400** [comment]
okay the only broken part is I need the compiled frontend folder

**Comment by om3ga6400** [comment]
what would that be

**Comment by om3ga6400** [comment]
@ColtNovak

**Comment by ColtNovak** [comment]
hmm idk

**Comment by ColtNovak** [comment]
you mean like the client?

**Comment by ColtNovak** [comment]
https://chat.codechat.tech/ is the client
